scripts/ci/in_container/entrypoint_ci.sh

```diff
@@ -211,6 +211,10 @@ fi
 
 TEST_DIR="tests/"
```

Comment:
```suggestion
TESTS_TO_RUN="tests/"
```

Reply:
I think this will be better name :)

Reply:
fixed!!! https://github.com/apache/airflow/pull/7396/commits/f6be34fd8725ae44b156f0e51076d92bf69a36a0